CIP-Improved_Rewards_Scheme_Parameters/README.md

```diff
@@ -0,0 +1,238 @@
+---
+CIP: ????
+Title: Improved Rewards Scheme Parameters
+Status: Proposed
+Category: Reward-Sharing Scheme
+Authors:
+    - Tobias Fancee <tobiasfancee@gmail.com>
+Implementors:
+    - Input Output Global
+    - Cardano Foundation
+    - Emurgo
+Discussions:
+    - https://forum.cardano.org/t/cip-improved-rewards-scheme-parameters/112409
+Created: 2022-01-03
+License: CC-BY-4.0
+---
+
+# Abstract
```

Comment:
```suggestion
# Improved Rewards Scheme Parameters
## Abstract
```
The H1 header level is generally used for a document title, while the top-level sections are H2's, and so on to H3 as those sections are broken up.  Therefore for proper outline you should add an H1 title here (duplicating the `Title:` header field if necessary).